src/process.cr

```diff
@@ -142,7 +142,7 @@ class Process
       # error:
       errno = Errno.value
       LibC.pthread_sigmask(LibC::SIG_SETMASK, pointerof(oldmask), nil)
-      raise Errno.new("fork", errno)
+      raise RuntimeError.from_errno("fork", errno)
```

Comment:
I think it would be kind of nice if `Process.new` on a non-existing file raised `File::NotFoundError`, i.e. all file-related errors were handled like this, even in mixed-function syscalls.